docs/API.md

```diff
@@ -556,3 +556,19 @@ hoistStatics(higherOrderComponent: HigherOrderComponent): HigherOrderComponent
 ```
 
 Augments a higher-order component so that when used, it copies static properties from the base component to the new component. This is helpful when using Recompose with libraries like Relay.
+
+### `createElement()`
+
+```js
+createElement(component: ReactElementType, props: Object, children: ReactChildrenType): ReactElementType
+```
+
+Creates a component similar to [`React.createElement`](https://facebook.github.io/react/docs/top-level-api.html#react.createelement), but safely handles assigning a key to a pure functional component by wrapping it in a class.
```

Comment:
I think you're confusing `createElement()` with `toClass()`. The purpose of the `createElement()` is to detect referentially transparent components and call them as functions rather than creating an unnecessary elements. See this doc for more details: https://github.com/acdlite/recompose/blob/master/docs/performance.md